examples/angular2/tsconfig.json

```diff
@@ -1,15 +1,13 @@
 {
   "compilerOptions": {
-    "target": "ES5",
+    "target": "es5",
     "module": "commonjs",
     "moduleResolution": "node",
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
-    "noImplicitAny": true
-  },
-  "files": [
-    "app/bootstrap.ts"
-  ]
+    "noImplicitAny": true,
+    "suppressImplicitAnyIndexErrors": true
+  }
```

Comment:
When inspecting the example source code all a developer should care about is the TS files, since the JS are generated by the compiler. Having them mixed is unnecessary noise in my opinion, and could potentially throw away a newbie developer maybe?

This is the approach taken for actual projects built with any tech stack that involves some sort of transpiling, so it would be beneficial for this example to reflect that.